exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -278,7 +277,7 @@ public boolean setSessionOption(String name, String value) {
    */
   public String registerTemporaryTable(AbstractSchema schema, String tableName) throws IOException {
       addTemporaryLocation((WorkspaceSchemaFactory.WorkspaceSchema) schema);
-      String temporaryTableName = Paths.get(sessionId, UUID.randomUUID().toString()).toString();
+      String temporaryTableName = new Path(sessionId, UUID.randomUUID().toString()).toUri().getPath();
```

Comment:
I wonder if we should use one of the many platform-independent path representations? We have the original Java `File`, the new Java `Path`, the Hadoop HDFS `Path`... If we use strings for paths, we run into the very problems that those packages are designed to address.

For example, do the above handle the "D:" directory suffix on Windows? Resolving "." and ".." in paths? The "~" indicator of use home folder? These all vary across platforms.